ha_mroonga.cpp

```diff
@@ -12245,7 +12247,10 @@ int ha_mroonga::generic_store_bulk_float(Field* field, grn_obj* buf)
 {
   MRN_DBUG_ENTER_METHOD();
   int error = 0;
-  double value = field->val_real();
+  double value = 0;
```

Comment:
```suggestion
  double value = 0.0;
```

`0`でも動くんだけど、`double`で宣言しつつ初期化しているので`double`なリテラルを使ったほうがベターです。

Reply:
たしかにです。ありがとうございます！